Sources/AppStoreConnectCLI/Commands/TestFlight/Sync/TestFlightProgramDifference.swift

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+import Model
+
+struct TestFlightProgramDifference {
+
+    enum Change {
+        case addBetaGroup(BetaGroup)
+        case removeBetaGroup(BetaGroup)
+        case addBetaTester(BetaTester, toBetaGroups: [BetaGroup])
+        case removeBetaTester(BetaTester, fromBetaGroups: [BetaGroup])
+
+        var description: String {
+
+            let operation: String = {
+                switch self {
+                case .addBetaGroup, .addBetaTester: return "added to"
+                case .removeBetaGroup, .removeBetaTester: return "removed from"
+                }
+            }()
+
+            switch self {
+            case .addBetaGroup(let betaGroup), .removeBetaGroup(let betaGroup):
+                let name = betaGroup.groupName ?? ""
+                let bundleId = betaGroup.app?.bundleId ?? ""
+
+                return "Beta group named: \(name) will be \(operation) app with bundleId: \(bundleId)"
+
+            case .addBetaTester(let betaTester, let betaGroups),
+                 .removeBetaTester(let betaTester, let betaGroups):
+                let email = betaTester.email ?? ""
+                let groupNames = betaGroups.map({ $0.groupName ?? "" }).joined(separator: ", ")
+
+                return "Beta Tester with email: \(email) will be \(operation) groups: \(groupNames)"
```

Comment:
What do you think of adding `bundleId` in this one? Otherwise, when two apps have two groups with the same name, this will become confusing.
```suggestion
                return "Beta Tester with email: \(email) will be \(operation) groups: \(groupNames) in bundleId: \(bundleId)"
```

Reply:
🤔 for this one there one there is not one bundleId there could be groups here across many apps
So it would need to be a list of group names and bundle ids e.g.

"Beta Tester with email: example@bla.com will be added to group1 in com.example.app1, group2 in com.example.app2"